Gruntfile.js

```diff
@@ -1,70 +1,64 @@
 module.exports = function(grunt) {
 
-	grunt.initConfig({
-		pkg: grunt.file.readJSON('package.json'),
-		jshint: {
```

Comment:
Yes, I was thinking the same thing - certainly no harm in them both running anyway.